bundle-workflow/src/test_workflow/perf_test_cluster.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import subprocess
+from contextlib import contextmanager
+
+from test_workflow.test_cluster import TestCluster
+
+
+class PerformanceTestCluster(TestCluster):
```

Comment:
class doesn't match filename

Reply:
Renamed the class name to `PerfTestCluster`